### DISCUSSION TITLE
Strange error

### DISCUSSION THREAD

**Discussion by Shomari-Healy** [EDmodel/ED2#407] [General]
Hi everyone,
I am getting a strange error when I try to run the model saying 'ERROR STOP IN LINKLIST: YOU CAN NOT USE AN ID TWICE: OUTCONFIG'. I have checked in the src but cannot find the issue. Outconfig is only present in ed_xml_config.f90 but I haven't changed anything in this file and there is nothing wrong with it as far as I can tell. Has anyone encountered this issue before/knows how to solve it? (I am trying to do a pv simulation)

I am happy to provide more files and information if you need.

[output.txt](https://github.com/user-attachments/files/17342697/output.txt)

**Comment by mpaiao** [comment]
@Shomari-Healy I wonder if this is a recent bug in the code, I merged a few pull requests recently, and my hunch is that there may be some duplicated variable name in subroutine `write_ed_xml_config` in `ed_xml_config.f90`. But maybe @mdietze can confirm if that would trigger this specific error.

**Reply by Shomari-Healy** [reply]
Right ok. When I checked the file nothing seemed duplicated but maybe I missed it. How do you suggest I go about solving this issue or is it something that would need to be solved on your end due to the pull requests?
